### DISCUSSION TITLE
Neovim plugin: `=copilot#TextQueuedForInsertion()`

### DISCUSSION THREAD

**Discussion by joelburget** [community/community#42883] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Sometimes when using the Neovim plugin, when I tab to accept it inserts `=copilot#TextQueuedForInsertion()` instead of the completion. Unfortunately I'm not sure how to repro consistently.

**Comment by iamcjwilliams** [comment]
@joelburget Please visit here to trouble your copilot issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot

**Comment by joelburget** [comment]
Thanks, but all this says is to use `:Copilot status`, which returns `Copilot: Enabled and online`. Not a whole lot of information. This issue is nondeterministic -- it happens only ~10-20 % of the time. I bet the status is `Enabled and online` throughout.

**Comment by jmiller-rise8** [comment]
Probably a bug here: https://github.com/github/copilot.vim/blob/c7d166ebda265370f38cec374e33f02eeec2f857/autoload/copilot.vim#L477.

I'll try to replicate tonight.

**Reply by jmiller-rise8** [reply]
Well the function exists. Could be key sequence was not entered correctly, or the mapping was not defined in the correct mode or buffer.

I would try manually executing the function call copilot#TextQueuedForInsertion() to see if it returns the expected result if when this happens again.

I'll continue trying to replicate later but I'm not having any success. Might help to know what language, nvim version, and OS you're using.
